src/HW5.java

```diff
@@ -0,0 +1,45 @@
+public class HW5 {// Первое задание
+    String name;
+    String position;
+    String phone_number;
+    String email;
+    int salary;
+    int age;
+
+    public HW5 (String name, String position, String email, String phone_number, int salary, int age) {
+        this.name = name;
+        this.position = position;
+        this.email = email;
+        this.phone_number = phone_number;
+        this.salary = salary;
+        this.age = age;
+    }
+
+    // Метод
+
+    public String toString(){
+        return String.format("Имя: %s \tДолжность: %s \nEmail: %s \t Номер телефона: %s \n Зарплата: %d \t Возраст: %d \n",
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.